artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/queue/UpdateQueue.java

```diff
@@ -31,7 +31,7 @@
    String name;
 
    @Option(name = "--deleteOnNoConsumers", description = "whether to delete when it's last consumers disconnects)")
-   Boolean deleteOnNoConsumers = null;
+   String deleteOnNoConsumers = null;
```

Comment:
You need to be creative with these names to avoid the -noNo case or course.  


It is the framework we have at the moment. 



Whatever you do it needs consistency.  


Talking about consistency the exceptions need to show help errors and these addresses are not working on that sense.  I will make some fixes next week.